packages/payment-integration-api/src/hosted-form/hosted-form.ts

```diff
@@ -0,0 +1,13 @@
+import { OrderPaymentRequestBody } from "../order";
+import { PaymentAdditionalAction } from "../payment";
+
+import { HostedInputSubmitSuccessEvent } from "./hosted-input-events";
+
+export interface HostedForm {
+    attach(): Promise<void>;
+    detach(): void;
+    getBin(): string | undefined;
```

Comment:
```suggestion
    getBin?(): string;
```

Reply:
ewww, the previous implementation was this. Hence it cannot only return `string` if I am not mistaken.
```
getBin(): string | undefined {
        return this._bin;
    }
```